app/views/users/new.haml

```diff
@@ -4,7 +4,7 @@
       %tr
         %th{colspan: 2} Sign Up
     %tbody
-      %tr#username
+      %tr#username-row
```

Comment:
I am reasonably confident despite not having checked that this will break the user validation javascript

Reply:
Can we call these like `#signup-username`, `#signup-password`, `#signup-secret` btw? That way we describe their uniqueness better. Just before we go changing the javascript.

Reply:
Can do

Reply:
Yep, user validation javascript will need tweaking: <https://github.com/Marri/glowfic/blob/9692cda6d8ecc3b53f1d7058d57d5cbc5714e812/app/assets/javascripts/users/new.js#L3>

Specifically, `#username`, `#email`, `#password`, and also most uses of `addAlertAfter`.